### PR TITLE
Clean ups of Sqlite test code

### PR DESCRIPTION
**Pull request by rvl**
Relates to issue #154 
Base branch is PR #373 

# Overview

- These were review comments on #259

### CODE REVIEW

File: lib/core/test/unit/Cardano/Wallet/DB/StateMachine.hs [hunk 1]
```diff
@@ -16,6 +16,26 @@
 
 {-# OPTIONS_GHC -fno-warn-orphans #-}
 
+-- |
+-- Copyright: © 2018-2019 IOHK
+-- License: MIT
+--
+-- These are state machine model tests for the DBLayer implementations.
```
Comment (jonathanknowles) [inline lib/core/test/unit/Cardano/Wallet/DB/StateMachine.hs]: For Haddock comments, if you write `'DBLayer'` (in single quotes), then Haddock will turn this into a hyperlink that jumps to the `DBLayer` type definition.

File: lib/core/test/unit/Cardano/Wallet/DB/StateMachine.hs [hunk 2]
```diff
@@ -164,6 +169,7 @@ fromMockPrivKey s = (k, Hash (B8.pack s))
 toMockPrivKey :: (Key purpose XPrv, Hash "encryption") -> MPrivKey
 toMockPrivKey (_, Hash h) = B8.unpack h
 
+-- | Mock representation of a DBLayer
```
Comment (jonathanknowles) [inline lib/core/test/unit/Cardano/Wallet/DB/StateMachine.hs]: Similarly to the comment above, would recommend turning this into a Haddock hyperlink, like so:
```hs
 -- | Mock representation of a 'DBLayer'.
```

**Review by jonathanknowles — COMMENTED** [review]
Looks good to me.

Just a small point w.r.t. to Haddock comments. If you write the name of a type in a Haddock comment, then putting it in single quotes will get Haddock to create a hyperlink that goes to type.

For example:
```hs
-- | A mock representation of the 'Wibble' data type.
```
(If Haddock can find a definition of `Wibble`, then the generated Haddock documentation will link it up.)

File: lib/core/test/unit/Cardano/Wallet/DB/StateMachine.hs (hunk 1 above)
Comment (KtorZ) [reply lib/core/test/unit/Cardano/Wallet/DB/StateMachine.hs]: :+1: Fixed.

File: lib/core/test/unit/Cardano/Wallet/DB/StateMachine.hs (hunk 2 above)
Comment (KtorZ) [reply lib/core/test/unit/Cardano/Wallet/DB/StateMachine.hs]: Fixed :+1:

**Review by KtorZ — APPROVED** [review]

**Pull request merged**